engine/access/rest/link_generator.go

```diff
@@ -0,0 +1,83 @@
+package rest
+
+import (
+	"github.com/gorilla/mux"
+
+	"github.com/onflow/flow-go/engine/access/rest/generated"
+	"github.com/onflow/flow-go/model/flow"
+)
+
+type LinkFun func(id flow.Identifier) (string, error)
```

Comment:
nit: 
```suggestion
type LinkFunc func(id flow.Identifier) (string, error)
```

`LinkFun` is amusing, but confusing